src/main/scala/com/google/cloud/genomics/spark/examples/GenomicsConf.scala

```diff
@@ -58,13 +58,18 @@ class GenomicsConf(arguments: Seq[String]) extends ScallopConf(arguments) {
   }
 
   def getReferences = {
-    this.references().split(",").map(contig => {
-      val data = contig.split(":")
-      (data(0), (data(1).toLong, data(2).toLong))
-    }).toMap
+    Contig.parseContigsFromCommandLine(this.references())
   }
 }
 
 class PcaConf(arguments: Seq[String]) extends GenomicsConf(arguments) {
   val numPc = opt[Int](default = Some(2))
+  val allReferences = opt[Boolean]()
```

Comment:
@deflaux - the `getContigsInVariantSet` method has a `excludeXY` boolean param which will do the filtering for you. (on line 71 in this file you guys are using it)

In dataflow, we set that param to true for most of our analyses so that the user doesn't have to worry. If a different analysis wanted XY to be included, it would just pass in a default of false for the user.